.github/workflows/call_graph_generator.yaml

```diff
@@ -0,0 +1,33 @@
+name: Call-Graph-Generator
+
+on:
+  pull_request:
+    branches: "master"
+
+jobs:
+  call_graph:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python 3.7.5
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.7
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install graphviz
+          pip install pydot
+          pip install pyan3==1.1.1
+      - name: Generate call graph
+        run: |
+          cd src/epyq
+          pyan3 **.py --uses --no-defines --colored --grouped --annotated --html > call_graph.html
+          mv call_graph.html ../../
```

Comment:
Would it make sense to combine these two lines? Something like this:
`pyan3 **.py --uses --no-defines --colored --grouped --annotated --html > ../../call_graph.html`

Reply:
No longer applicable.